### ISSUE TITLE
Χρεμώς to delete

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#760]
Dear Magdalena,
the file is to delete but I cannot: why?
Could you please delete it?
Thank you very much,
best,
Sophie
![Capture d’écran 2024-01-17 à 19 11 14](https://github.com/eXistSolutions/LGPN-ling/assets/16923546/d45ab5fc-87bd-4d2d-8e8e-7a6500f49343)

**Comment by Hippodamie** [comment]
Also, please, could you delete this file?
Thank you very much,
best,
![Capture d’écran 2024-01-17 à 19 19 53](https://github.com/eXistSolutions/LGPN-ling/assets/16923546/a4dd063b-09a2-4996-98aa-5d548d54ed6d)

Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,
thank you for what you did to fix the servor this week-end!
I have, please, 3 more deletions of files to ask for: Hypeiron; Peithino; Heroboulos (doublet: we just keep one).
Thank you for that,
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena: another one to delete, please: Klēsirodos (saving Klēsirhodos).
Μany thanks,
all best,
Sophie

**Comment by Hippodamie** [comment]
Dear Magalena,
another one to delete: Etymōndēs.
Many thanks for that but it is a pity for us not to be able to do it by ourselves. And it will happen for years...
All best,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,
hope you are well.
Could you please delete one more: Philekleēs?
Thank you very much,
all best,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,
please, could you delete Cleōnēs file?
Many thanks,
all best,
Sophie

**Comment by Hippodamie** [comment]
Deat Magdalena,
anther one to delete: Pankōn2.
Thank you in advance,
all best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Magdalena,
one more: Koidrōn...
Thank you in advance,
all best,
Sophie

**Comment by tuurma** [comment]
> Dear Magdalena: another one to delete, please: Klēsirodos (saving Klēsirhodos). Μany thanks, all best, Sophie

I didn't remove as it was edited quite recently, please confirm if I should?

<img width="1417" alt="image" src="https://github.com/eXistSolutions/LGPN-ling/assets/449468/3df35e6e-79ff-4f73-9040-be4408360409">


https://lgpn-ling.huma-num.fr/index.html?filter=Kle%CC%84sirodos&filterBy=name

**Comment by tuurma** [comment]
> Pankōn

There is only one Pankon, edited in March, so I kept it for the moment

<img width="1395" alt="image" src="https://github.com/eXistSolutions/LGPN-ling/assets/449468/a6da42a6-1e2b-4c82-8465-798fa3c41fe3">


https://lgpn-ling.huma-num.fr/index.html?filter=%CE%A0%CE%B1%CE%B3%CE%BA%E1%BF%B6%CE%BD&filterBy=name

**Comment by tuurma** [comment]
> Dear Magdalena, please, could you delete Cleōnēs file? Many thanks, all best, Sophie

I only found Kleones, not Cleones, please confirm, should I remove it?

<img width="1395" alt="image" src="https://github.com/eXistSolutions/LGPN-ling/assets/449468/c194fbd5-9fca-4b13-b90c-c0e795b9ea95">

https://lgpn-ling.huma-num.fr/index.html?filter=Kleone*&filterBy=name

**Comment by tuurma** [comment]
Dear Sophie, above I asked for clarification for 3 names. Under your original requests I put signs, thumbs up where I just did it, and "eyes" where I made a comment and need your confirmation.

**Comment by Hippodamie** [comment]
Dear Magdalena, yes, please !Le 22 mars 2024 à 13:54, Magdalena Turska ***@***.***> a écrit :﻿

Dear Magdalena, please, could you delete Cleōnēs file? Many thanks, all best, Sophie

I only found Kleones, not Cleones, please confirm, should I remove it?
image.png (view on web)
https://lgpn-ling.huma-num.fr/index.html?filter=Kleone*&filterBy=name

**Comment by Hippodamie** [comment]
Yes, please!Thank you (vert often there ar doublets, and one of them to delete.All best,SophieLe 22 mars 2024 à 13:47, Magdalena Turska ***@***.***> a écrit :﻿

Dear Magdalena: another one to delete, please: Klēsirodos (saving Klēsirhodos). Μany thanks, all best, Sophie

I didn't remove as it was edited quite recently, please confirm if I should?
image.png (view on web)
https://lgpn-ling.huma-num.fr/index.html?filter=Kle%CC%84sirodos&filterBy=name

**Comment by Hippodamie** [comment]
One more: Theukylidēs exactly.
Thank you,
all best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Magdalena, à new one, please, to delete: Kōmadēs.
Many thanks,
best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Magdalena,
one more deletion of name to operate: Κόρων Kōrōn. A doublet of another one (Κόρων Korōn, to keep). 
Thank you in advance,
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
A small lot to delete:
Κοπετίων Kopetiōn
Κορήχα, Korēcha
Κοττίων Kottiōn
Καρταπονως Kartaponōs
Νεμοκράτης Nemokratēs
Κρατήσα Kratēsa
Κράννων? Κραννο̄ν?
Thank you in advance,
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,
in the previous lot, read for the last translitterated: Krannōn.
Could you please delete another doblet too: Ktesarete (the almost empty file; we keep the other one, with full commentary).
Thank you for all,
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,

could you please delete Thektamenes?
Thank you very much,
all best,
Sophie
